source/features/tab-to-indent.tsx

```diff
@@ -14,7 +14,8 @@ void features.add({
 	screenshot: 'https://user-images.githubusercontent.com/1402241/33802977-beb8497c-ddbf-11e7-899c-698d89298de4.gif'
 }, {
 	include: [
-		pageDetect.hasCode
+		pageDetect.hasCode,
```

Comment:
I don't think `hasCode` is necessary anymore
```suggestion
```

Reply:
Is code is needed for editing files etc.. But my first commit was correct. Not the second. Since all hasRichTextEditor adds is isNewIssue

Reply:
Editing files don’t use the rich text editor